ssh/ci/ssh.rake

```diff
@@ -0,0 +1,64 @@
+require 'ci/common'
+
+def ssh_version
```

Comment:
we don't appear to need either `ssh_version` or `ssh_rootdir` in this particular check.